### PR TITLE
Use Docker 1.13 RC

### PR DESCRIPTION
**Pull request by hjacobs**
To fix "Docker hanging" issue (https://github.com/docker/docker/issues/28889).

### CODE REVIEW

File: cluster/userdata-worker.yaml [hunk 1]
```diff
@@ -102,7 +138,28 @@ coreos:
         Type=oneshot
         ExecStart=/usr/bin/docker pull registry.opensource.zalan.do/teapot/microscope:v0.0.2
 
+
+ 
 write_files:
+
+  - path: "/etc/profile.d/docker-path.sh"
+    content: |
+      PATH="/opt/docker:$PATH"
```
Comment (mikkeloscar) [inline cluster/userdata-worker.yaml]: This file doesn't seem to be used?

File: cluster/userdata-worker.yaml [hunk 2]
```diff
@@ -102,7 +138,28 @@ coreos:
...
+
+  - path: "/etc/profile.env"
+    content: |
+      PATH="/opt/docker:/usr/sbin:$PATH"
```
Comment (mikkeloscar) [inline cluster/userdata-worker.yaml]: is `/usr/sbin` not in `$PATH`, if not, why do we need it?

File: cluster/userdata-master.yaml [hunk 3]
```diff
@@ -113,6 +148,26 @@ coreos:
         ExecStart=/usr/bin/docker pull registry.opensource.zalan.do/teapot/microscope:v0.0.2
 
 write_files:
+
+  - path: "/etc/profile.d/docker-path.sh"
+    content: |
+      PATH="/opt/docker:$PATH"
```
Comment (mikkeloscar) [inline cluster/userdata-master.yaml]: This file doesn't seem to be used? (same as for the worker userdata).

File: cluster/userdata-worker.yaml (hunk 2 above)
Comment (Raffo) [inline cluster/userdata-worker.yaml]: we need it for iptables.

File: cluster/userdata-master.yaml (hunk 3 above)
Comment (szuecs) [inline cluster/userdata-master.yaml]: @mikkeloscar there is a for loop on /etc/profile.d/*.sh to source all of them

File: cluster/userdata-worker.yaml (hunk 1 above)
Comment (mikkeloscar) [inline cluster/userdata-worker.yaml]: @szuecs Ok, but the file below that seem to do the same thing is sourced in the service with `EnvironmentFile=/etc/profile.env`, so I assume this one isn't needed.

**Comment by szuecs** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

File: cluster/userdata-worker.yaml [hunk 4]
```diff
@@ -19,7 +19,43 @@ coreos:
...
+        LimitCORE=infinity
+        # Uncomment TasksMax if your systemd version supports .
+        # Only systemd 226 and above support this version.
+        TasksMax=infinity
+        TimeoutStartSec=0
+        # set delegate yes so that systemd does not reset the cgroups of docker containers
+        Delegate=yes
+
+        [Install]
+        WantedBy=multi-user.targe
```
Comment (mlafeldt) [inline cluster/userdata-worker.yaml]: `multi-user.target` 😄

Comment (hjacobs) [inline cluster/userdata-worker.yaml]: Uups, thanks!

**Pull request merged**